configs/mirrors.txt

```diff
@@ -1,14 +1,11 @@
 url='http://resources.ovirt.org/pub',whitelist=True
-url='http://mirror.linux.duke.edu/ovirt/pub'
```

Comment:
this one is still reachable but outdated, perhaps the syncing is not working anymore?

Reply:
Same as for gatech - no replies from them